ckan/adx_config.ini

```diff
@@ -121,15 +121,15 @@ ckan.redis.url = redis://redis:6379/1
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
 # Note: Plugins to the left take precendence over plugins to the right!!! Opposite to what you might expect.
-ckan.plugins = external_storage unaids  emailasusername restricted authz_service versions stats text_view image_view
+ckan.plugins = scheming_datasets external_storage unaids emailasusername restricted authz_service versions stats text_view image_view
```

Comment:
why do you need to change the order of the `scheming ` plugin?
Also could you fix the formatting of this config value? 1st line if super long while the last two are much shorter then the others.

Reply:
This was the only way unaids could override the scheming `upload.html` template here: https://github.com/fjelltopp/ckanext-unaids/pull/105/files#diff-8e5ca759def387b362229130f1827669ae72fe441c9884c2c4a59a617b3fd295

My override wasn't working until I moved scheming to before unaids in `ckan.plugins`

Reply:
@tomeksabala - After our call this morning, can you confirm this is ok now?

Reply:
Only this last bit: >Also could you fix the formatting of this config value? 1st line if super long while the last two are much shorter then the others.

Reply:
Done